src/core/connection.h

```diff
@@ -852,7 +863,27 @@ QuicConnAddOutFlowBlockedReason(
     _In_ QUIC_FLOW_BLOCK_REASON Reason
     )
 {
+    CXPLAT_DBG_ASSERTMSG(
```

Comment:
Thanks for adding this. Makes me wonder if we could do some SAL magic to make this evaluate at build time...